client/components/tinymce/plugins/wpcom/wpcom-utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Removes empty spaces from empty paragraphs
+ * This logic is borrowed from core's TinyMCE Plugin
+ *
+ * @see https://core.trac.wordpress.org/browser/trunk/src/wp-includes/js/tinymce/plugins/wordpress/plugin.js#L133
+ *
+ * @param  {String}   content TinyMCE Editor content
+ * @return {String}           Content with strings removed from empty paragraphs
+ */
+export function removeEmptySpacesInParagraphs( content ) {
+	return content.replace( /<p>([^<>]+)<\/p>/gi, function( tag, text ) {
```

Comment:
@azaozz do you know what the purpose of the `[^<>]` part is for? What are we trying to match specifically? I think it works fine, but the `>` jumps out at me.